utils/composer-packages/src/Rector/AddPackageVersionRector.php

```diff
@@ -56,29 +56,27 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if (! property_exists($node, 'stmts')) {
```

Comment:
This will actually hide the type to both PhpStorm and PHPStan.

It's better to use instanceof of type check, ideally in strict form. That way both these tools (and Rector :wink: ) can understand context and warn about bugs and refactor :+1: